play/src/front/Components/MyCamera.svelte

```diff
@@ -13,20 +13,28 @@
     import { LL } from "../../i18n/i18n-svelte";
     import { inExternalServiceStore } from "../Stores/MyMediaStore";
     import { localUserStore } from "../Connection/LocalUserStore";
+    import { streamableCollectionStore } from "../Stores/StreamableCollectionStore";
     import SoundMeterWidget from "./SoundMeterWidget.svelte";
     import { srcObject } from "./Video/utils";
     import Woka from "./Woka/WokaFromUserId.svelte";
     import microphoneOffImg from "./images/microphone-off.png";
     import cameraOffImg from "./images/camera-off.png";
 
     let stream: MediaStream | null;
+    let videoElement: HTMLVideoElement;
     let userName = localUserStore.getName();
     let backgroundColor = Color.getColorByString(userName ?? "default");
     let textColor = Color.getTextColorByBackgroundColor(backgroundColor);
 
+    let aspectRatio = 1;
+
     const unsubscribeLocalStreamStore = localStreamStore.subscribe((value) => {
         if (value.type === "success") {
             stream = value.stream;
+            // TODO: remove this hack
+            setTimeout(() => {
+                aspectRatio = videoElement != undefined ? videoElement.videoWidth / videoElement.videoHeight : 1;
+            }, 100);
```

Comment:
To avoid these setTimeout, you can trigger the aspectRatio computation as soon as the first frame is received (using `requestVideoFrameCallback()`).

I think something like this might do the trick:

```suggestion
            if (videoElement) {
                videoElement.on("requestVideoFrameCallback", () => {
                    aspectRatio = videoElement != undefined ? videoElement.videoWidth / videoElement.videoHeight : 1;                
                });
            }
```

Reply:
have you tested with mobile definition into the listner zone and bubble connection?